NickvisionTubeConverter.Shared/Models/MediaUrlInfo.cs

```diff
@@ -83,18 +83,20 @@ private MediaUrlInfo(string url)
                     }
                     if (mediaInfo.HasKey("entries"))
                     {
+                        var i = 1u;
                         foreach (var e in mediaInfo["entries"].As<PyList>())
                         {
                             if (e.IsNone())
                             {
                                 continue;
                             }
-                            mediaUrlInfo.ParseFromPyDict(yt, e.As<PyDict>(), true, url);
+                            mediaUrlInfo.ParseFromPyDict(yt, e.As<PyDict>(), i, url);
+                            i++;
```

Comment:
I'm not entirely sure about this, so let me explain. Let's say we have 3 items playlist with the 2nd one being unavailabe. With this code, available items will be numbered 1 and 2 (in track metadata and when numbering titles), this is the old behaviour and that's why I kept it.
But I can change it so numbers will represent the actual position of media in playlist, so in this example items will be numbered 1 and 3.
I'm not sure what is better.